test/teslamate/import_test.exs

```diff
@@ -120,6 +120,7 @@ defmodule TeslaMate.ImportTest do
                address_id: nil,
                cost: nil,
                duration_min: 70,
+               charge_rate: decimal(123),
```

Comment:
I wonder if 123 is a realistic charge rate

Reply:
It is, if configured in km it would be an equivalent ~17kW

Reply:
Then `charge_rate` is not a self explaining variable name. `duration_min` is clear, but `charge_rate` is not.

Reply:
It's the rate in distance/time et which you're effectively charging, it has the same name in the API. It's the number that gets displayed when you switch your cars battery level to distance.